### DISCUSSION TITLE
The reliability of today's firmware.

### DISCUSSION THREAD

**Discussion by dgeo20** [meshtastic/Discussions#55] [General]
Hello, community!

I currently have two solar RAKs in router mode, operating autonomously for almost two years without any on-site intervention. 

I would like to update them to version 2.5.

The latest versions often tend to disrupt networks.

Over the years, we have gained functionality but lost reliability.

I compare Discord to a black hole: a posted message disappears just as quickly into the crowd.

Regarding the use of GitHub as a discussion forum, this deserves a separate topic.

I see many networks disappearing or remaining on 2.4 in my region.

How can one still trust the MeshTastic software for crisis situation networks?

A big thank you to the developers of the project, I sincerely mean it. But in which direction is the captain heading?

**Comment by ikearat** [comment]
What you said about "black hole" is how I feel about support. Unless you find somebody on Reddit or a random u-toob video, there is no one handling the "customer facing" part of the project. Even if the support person can only say "I will talk to the programmer(s)" it would help keep people like myself form looking for other projects to get interested in. Example: Never, never ever read the "issues" tab for any of the Meshtastic projects. You will get the strong feeling that when a problem gets a little difficult, the discussion dies-right-there with no solution... and then a new unrelated feature pops up in the firmware or we get a screwed-around user interface in Android. Like, why does "Remote Administration" on a total stranger's node show "active" fields only to show "No channel" when you select a Configuration item? There should be no active fields at all, because there is no channel. a-b thinking.

**Reply by garthvh** [reply]
There is a ton of well written documentation with great search. The expectation that you are going to get free personalized support is going to lead to frustration.

https://meshtastic.org/

**Reply by ikearat** [reply]
I understand what you said. I don't expect "free" support, on that, and out of curiosity how would one acquire paid support?
The problem is not about personal support, it's about support at all. Many clearly written examples of systemic issues go without response. This is why postings like this and my follow up happen. Searching does not produce solutions to problems that are not being handled. There is a thread here on the Meshtastic Github Discussions where a careful experiment by a user is exposing a communication issue between three nodes all within the user's control. No one is responding with anything other than 'go search the documentation'. It's not fixing the problem. All of us have "expectations" and we are willing to try recommend solutions but as far as I can see, few of the recommendations are coming from the developers. Fortunately, the cost of entrance into Meshtastic is low and unless you jumped-in with unrealistic expectations, or were "here" from the start and have invested in a lot of infrastructure over time (and watched your system degrade into instability), you can just give up and go find something else to spend your 1st World spare time on. As for myself, and for a little while longer, I will hang in and see if there is any stability in the short future. Then, and you probably will welcome this, I'm 'outta here'.

**Reply by garthvh** [reply]
Pretty clearly you expect some sort of completeness in user discussions that will never exist. Most of these "experiments" expect meshtastic to behave like TCP, which will never be the case with a low bandwidth flooded mesh. Seriously, read the docs not discussions from other confused users, which just multiplies confusion.

**Reply by garthvh** [reply]
You can pay for support at meshtastic.com

**Reply by ikearat** [reply]
email sent. I asked if there is a support level that is intended for people who are not developing hardware or using Meshtastic for commercial applications.

**Reply by ikearat** [reply]
I received an almost immediate reply! Please read the copy below. I got a question to my question and would like advice from the forum so I can make an intelligent and informed reply.
--- begin copy ---
Good day [redacted],

What sort of support are you looking for? This is not something we currently offer but the concept does have merit.



On Sun, Nov 10, 2024 at 12:14 PM '[meshtastic.com](http://meshtastic.com/)' via Meshtastic Contact <[contact@meshtastic.com](mailto:contact@meshtastic.com)> wrote:
Email:
[redacted my email address]

Message:
A user on the Github Discussions Forum provided me with this link (leads to here) in response to me asking if there was paid support for Meshtastic users. This individual uses the name of Garth Vander Houwen. I see that your support is catering to organizations and hardware developers. What level of participation, paid support do you recommend for an individual Meshtastic user who only intends to use provided hardware and software and not bring to market products or use Meshtastic for commercial purposes?


-- 
JM Casler | [jm@meshtastic.com](mailto:jm@meshtastic.com) | +1 408 806 8098
Meshtastic | Project Lead & President | [meshtastic.org](http://meshtastic.org/)
Meshtastic Solutions | CEO | [meshtastic.com](http://meshtastic.com/)
--- end copy ---

**Reply by caveman99** [reply]
What do you expect to happen by posting a private conversation on here? MC clearly stated we are thinking about this. Please take a step back and answer the question from the Mail. "What sort of support are you looking for?" Instead you post it up in public just to rant about it. I get the slight suspicion you don't need support but just wanna prove a point.

**Reply by adingbatponder** [reply]
I think that many private people (not businesses) are keen to hear expert answers to simple, almost "FAQ" type questions and would consider asking meshtastic.com : but the above response means we need not bother them if we are not a business I guess.

**Reply by rcarteraz** [reply]
> I think that many private people (not businesses) are keen to hear expert answers to simple, almost "FAQ" type questions and would consider asking meshtastic.com : but the above response means we need not bother them if we are not a business I guess.

We both know that isn't what is being said. You are very aware there's multiple avenues available for "FAQ type questions", you are just not a fan of several of them and mad the Discourse was shuttered. That's fine, but don't try and intentionally misrepresent the conversation. The perception from the original post was potentially seeking more specialized or guaranteed support beyond what’s typically offered for free, and that’s why potential paid options were mentioned. If that's not what you're looking for, there's absolutely nothing wrong with that, by all means make use of the available avenues (of which you've made known you don't care for most).

**Comment by dgeo20** [comment]
Meshtastic updates work like an endless loop. 

With each version, there’s an improvement, followed by a bug fix to address issues introduced by that improvement, which will eventually become a real improvement only three versions later after all bugs are fixed. 

One morning, during the week, you receive a notification on your Android app: you can no longer manage your node. 
Please update the firmware.

I find other users' discussions very useful for the project. The documentation doesn’t always solve "phantom issues."

The Meshtastic project lacks LTS versions: 3 months of bug fixes, followed by 6 months of validation.

**Reply by garthvh** [reply]
Those are requirements that need to be funded.

**Reply by mc-hamster** [reply]
> Meshtastic updates work like an endless loop.
> 
> With each version, there’s an improvement, followed by a bug fix to address issues introduced by that improvement, which will eventually become a real improvement only three versions later after all bugs are fixed.
> 
> One morning, during the week, you receive a notification on your Android app: you can no longer manage your node. Please update the firmware.
> 
> I find other users' discussions very useful for the project. The documentation doesn’t always solve "phantom issues."
> 
> The Meshtastic project lacks LTS versions: 3 months of bug fixes, followed by 6 months of validation.

Would you like to take lead on doing this?
